extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js

```diff
@@ -9,7 +9,8 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 
 	var dfpId = '5441',
 		megaSlots = [
-			'INVISIBLE_SKIN'
+			'INVISIBLE_SKIN',
+			//ToDo: BLB Mega global - 'BOTTOM_LEADERBOARD'
```

Comment:
Add ticket number please (create one for clean up if it doesn't exist)

Reply:
Done.